Rlabkey/NEWS

```diff
@@ -1,3 +1,6 @@
+Changes in 3.4.4
+  o Update documentation for experiment.createRun to include an example of inserting run level properties
```

Comment:
actually, I never published v3.4.3 after Ben B's last change, so you can just include this update with that last version and revert the version number change in Rlabkey/DESCRIPTION.